c2rust-analyze/src/main.rs

```diff
@@ -274,10 +276,15 @@ fn is_build_script(at_args: &[String]) -> anyhow::Result<bool> {
 
 /// Run as a `rustc` wrapper (a la `$RUSTC_WRAPPER`/[`RUSTC_WRAPPER_VAR`]).
 fn rustc_wrapper() -> anyhow::Result<()> {
-    let mut at_args = env::args().skip(1).collect::<Vec<_>>();
+    let no_cargo = env::var_os(NO_CARGO_VAR).is_some();
```

Comment:
Since you're using this from two places, maybe put it in a helper function?

Reply:
I added a helper function and put a little description of the behavior in its doc comment.